packages/jumpstarter-driver-corellium/jumpstarter_driver_corellium/client.py

```
@@ -0,0 +1,20 @@
+from contextlib import contextmanager
```

Ruff (0.8.2)

`contextlib.contextmanager` imported but unused

Remove unused import: `contextlib.contextmanager`

(F401)